src/test/java/ItemListTest.java

```diff
@@ -0,0 +1,222 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ItemListTest {
+
+    @Test
+    @DisplayName("When input is Empty")
+    void checkIfInputEmpty() {
+        ItemList item = new ItemList();
+        assertEquals(false,item.isInputValid(""));
```

Comment:
with assertion, you can pass message also for user readability.
follow : 
http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertArrayEquals(java.lang.String,%20byte[],%20byte[])